python/ray/_private/log_monitor.py

```diff
@@ -140,6 +140,7 @@ def update_log_filenames(self):
         # runtime_env setup process is logged here
         runtime_env_setup_paths = glob.glob(
```

Comment:
Another thing we can do is update the list of files scanned more infrequently.

Reply:
Oh yeah actually that sounds good. I think we can just do that in this pr?

Reply:
Sure, it sounds good.

Reply:
Update every 0.5 second if there are more than 1000 files now.